handlers.go

```diff
@@ -32,16 +32,19 @@ func mainHandler(w http.ResponseWriter, r *http.Request) {
 	var falcopayload types.FalcoPayload
 
 	stats.Requests.Add("total", 1)
+	countMetric("total", 1, []string{})
 
 	if r.Body == nil {
 		http.Error(w, "Please send a valid request body", 400)
 		stats.Requests.Add("rejected", 1)
+		countMetric("rejected", 1, []string{"error:nobody"})
 		return
 	}
 
 	err := json.NewDecoder(r.Body).Decode(&falcopayload)
 	if err != nil && err.Error() != "EOF" || len(falcopayload.Output) == 0 {
 		http.Error(w, "Please send a valid request body : "+err.Error(), 400)
+		countMetric("rejected", 1, []string{"error:invalidjson"})
 		stats.Requests.Add("rejected", 1)
```

Comment:
I know it's cosmectic, but can you keep the order of lines like that : `stats.Requests.Add()` and then `countMetric()`.